massrobotics_amr_sender_py/README.md

```diff
@@ -2,30 +2,48 @@
 
 Configuration-based ROS2 package for sending MassRobotics [AMR Interop Standard messages](https://github.com/MassRobotics-AMR/AMR_Interop_Standard) to complaint receivers.
 
-## Running from source
+# Installing
+
+## From binary packages
+
+The node is available as a released package and can be added manually to your ROS2 build installation:
+
+```sudo apt-get install ros-foxy-massrobotics-amr-sender```
+
+Or listed as a rosdep dependency and installed with `rosdep update`
+
+## Building from source
 
 Make sure `ros2` is installed properly. Clone this repository inside your `src` folder on your local workspace and build the package:
 
 ```bash
```

Comment:
Does `bash` work on markdown ? I always used `console`

Reply:
it seems to work